path_to_sim_points: "/home/anaritapereira/Dropbox/ladybug_sim/points2D/"
num_frames: 4084
